alibuild_helpers/build.py

```diff
@@ -344,33 +344,29 @@ def doBuild(args, parser):
                   "  git pull --rebase\n",
                   pwd=os.getcwd(), star=star()))
 
-  # Prefetch git heads in parallel
-  debug("Prefetching git heads in parallel:")
-
-  fetchers = list(filter(lambda p: "source" in specs[p], buildOrder))
-  poolConfig = {"processes": min(max(len(fetchers), 1), 16)}
-  pool = ThreadPool(**poolConfig)
-  debug("Created thread pool with config: %s" % poolConfig)
+  # Clone/update repos
+  reposWithSource = list(filter(lambda p: "source" in specs[p], buildOrder))
+  if args.offline:
+    banner("Offline mode - not attempting to update any repositories\n")
```

Comment:
I would not be so verbose here and change the style of the output:

```
debug("Not fetching repository updates")
```

will do :-)